.github/workflows/tests.yml

```diff
@@ -21,8 +21,79 @@ on:
       - 'doc/**'
 
 jobs:
+  no-meta:
+    strategy:
+       matrix:
+          OS: [macos-10.15, ubuntu-20.04]
+          CXXCOMPILER: [g++, clang++]
+    runs-on: ${{ matrix.OS }}
+
+    steps:
+      - name: Homebrew 
+        if: runner.os == 'macOS'
+        run: |
+           brew update
+           brew upgrade
+           brew install gsl
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Conan
+        run: |
+          pip3 install --user conan
+          export PATH=$(python3 -c 'import site; print(site.USER_BASE + "/bin")'):$PATH
+          conan remote add conan-hep https://api.bintray.com/conan/expander/conan-hep --force
+          conan install . --build=missing
+
+      - name: Configure
+        run: |
+          ./configure \
+             --with-cxxflags="-std=c++14 -fPIC" \
+             --with-cxx=$CXXCOMPILER \
+             --with-optional-modules=test \
+             --with-loop-libraries=collier,looptools \
+             --with-install-dir=install \
+             --disable-meta
+          make showbuild
+        env:
+          CXXCOMPILER: ${{ matrix.CXXCOMPILER }}
+
+      - name: Make
+        run: make -j2
+
+      - name: Meta tests
+        run: make -j2 execute-meta-tests
```

Comment:
I think we can leave out this line, as the meta code is disabled here.

Reply:
Good point. I've copy-pased without thinking.